web/spacewalk-web.changes.raul.update_rhnwebversion_check_migrationpaths

```diff
@@ -0,0 +1 @@
+- Update the version in the WebUI
```

Comment:
Not needed, if there are other changes in this package. Please check.

Reply:
As it was in previous changelogs, it will stay, but mentioning which version it is.